extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/PrometheusEmitter.java

```diff
@@ -91,6 +95,13 @@ public void start()
       } else {
         pushGateway = new PushGateway(address);
       }
+      exec = ScheduledExecutors.fixed(1, "PrometheusPushGatewayEmitter-%s");
```

Comment:
The executor should also be shutdown, maybe in `close`?

Reply:
> The executor should also be shutdown, maybe in `close`?

Yes, I almost forgot